src/gui/CMakeLists.txt

```diff
@@ -3,6 +3,12 @@ set(QTLIBS Qt5::Network Qt5::Widgets)
 include_directories(..)
 qt5_add_resources(NEOVIM_RCC_SOURCES data.qrc)
 if(WIN32)
+	if ($ENV{QT_STATIC})
```

Comment:
Why are you using an environment variable here? Instead can you set it as a cmake option and use something like `-DUSE_STATIC_QT=1` instead?

Otherwise you might end up mixing up build option if you forget to set the variable.

Reply:
ok, learning.

Reply:
Done